test/EpicTest.java

```diff
@@ -22,8 +17,9 @@ void add() {
         taskManager.getEpicById(1);
         assertEquals(taskManager.getAllEpics().size(),  1);
```

Comment:
В assertEquals сначала передается ожидаемое значение - 1, а затем актуальное. 
Должно быть наоборот: 
```suggestion
        assertEquals(1, taskManager.getAllEpics().size());
```